Assets/Scripts/Model/Core.cs

```diff
@@ -169,5 +170,29 @@ internal Player GetPlayer(int x, int y)
             }
             return null;
         }
+
+        private TurnMode lastturnmode;
+        public TurnMode LastTurnMode {
```

Comment:
Adjust this to:
`public TurnMode LastTurnMode { get; set; }`
and drop the private field. This implementation is unecessarily verbose, and the debug.log isn't necessary in a property.